ecommerce-ui/src/App.test.js

```diff
@@ -0,0 +1,9 @@
+import React from 'react';
+import ReactDOM from 'react-dom';
+import App from './App';
+
+it('renders without crashing', () => {
+  const div = document.createElement('div');
+  ReactDOM.render(<App />, div);
+  ReactDOM.unmountComponentAtNode(div);
+});
```

Comment:
Would like to see some examples of snapshot testing. You can review the in class code examples for where to start:
https://github.com/modern-web-application-uw19/react-testing/blob/master/src/App.test.js

The key point being matching the current render to a saved snapshot: 
`const wrapper = shallow(
      <Card title="News" description="lorem ipsum" />
    );
    expect(wrapper).toMatchSnapshot();`